sql-update/sick-prank.sql

```diff
@@ -0,0 +1,2 @@
+update "films"
+  set "rating" = 'G';
```

Comment:
This one isn't correct. It will set _every_ film to 'G', not just the 'NC-17' ones.